pages/common/dotnet-build.md

```diff
@@ -13,7 +13,7 @@
 
 - Compile in release mode:
 
-`dotnet build`
+`dotnet build --configuration Release`
```

Comment:
Missing token syntax?
```suggestion
`dotnet build --configuration {{Release}}`
```

Reply:
Are you sure about it? `Release` is one of possible configurations, so it's more like `dotnet build --configuration {{Configuration}}`, but then we need to change description too. @navarroaxel what do you think?

Reply:
I agree with @sbrl because we had this format in other pages. We have rules, but we have exceptions too. And the argument option should be between `{{}}`.